### ISSUE TITLE
Cataclysm-tiles.exe "has stopped working" when I load save game

### ISSUE THREAD

**Issue by KelrickLander** [CleverRaven/Cataclysm-DDA#22066]
Game version: 0.C-24240-gd6cff78

Operating system: Window 10 64bit

Tiles or curses: Tiles

Mods active: None

### Expected behavior
Save game to load successfully
### Actual behavior
Cataclysm-tiles.exe "has stopped working"
### Steps to reproduce the behavior
[Failed to Load.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/1350502/Failed.to.Load.zip)
[save.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/1350501/save.zip)
[Failed to load 2.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/1350500/Failed.to.load.2.zip)

I am using no mods at all, just the base dda mod.

When I load a clean save game and leave my farm to complete a quest at the refugee center, then return to my farm and save, the game will not load again.  When I try to load this new save game it pops a window up and says Cataclysm-tiles.exe "has stopped working".

My clean save file is attached.  I have also attached two different save files that fail after completing two different quests and trying to save near my farm.

I just want to know what is causing my save games to fail so that I can continue to enjoy this game.

Is there a way to view what is happening while the game is loading the save file so I can see where it is failing?  If it is some item, or too many items in the reality bubble I can fix that.  I just don't know what is happening.

**Comment by Nhatorama** [comment]
I had a similar problem, #21866 and my solution was to delete: 
G:\Cataclysm Dark Days Ahead Launcher Files\config\options.json

So delete options.json and then launch CDDA with the failed to load save.

**Comment by ejseto** [comment]
You can salvage your save file by opening up the file "o.-1.0" and modifying the npcs line at the bottom. Delete everything within the brackets, making it read:
>,"npcs":[]

You'll lose the npcs that were on that map but it's better than nothing.

There's some problem in overmapbuffer::fix_npcs, or at least that's where the symptom shows up. On the line
>npc &np = **it;

*it can be a null pointer, resulting in a seg fault.

**Comment by KelrickLander** [comment]
I had already tried deleting the option.json file an that had no affect.

**Comment by KelrickLander** [comment]
I did notice that when I would save my game near the Refugee Centre 2 of my NPCs would disappear.  They would usually reappear once I moved my reality bubble away from them and then come back.  Once they were back in the reality bubble they would come back to me.

I also noticed that the Old Guard Rep is missing from the Refugee Centre even though he is listed on the over map.

I will first try just removing some of the NPCs and see if that will fix it.  There are like 15 or more NPCs in that area.
